src/views/Benchmark/index.jsx

```diff
@@ -40,6 +40,7 @@ class Benchmark extends Component {
         <Graphs
           benchmarkData={benchmarkData}
           overviewMode={this.props.benchmark === 'overview'}
+          platform={this.props.platform}
```

Comment:
You can pass `benchmark` in here.

Reply:
> You can pass `benchmark` in here.

Value of benchmark is "overview" not the name/key of the benchmark.

Reply:
Only when you first land on the page.
If you select a benchmark from the dropdown menu you will see it will get a different value.